### DISCUSSION TITLE
Change !pb command to show hours in front

### DISCUSSION THREAD

**Discussion by tmgdejong** [runelite/runelite#15120] [Q&A] [answered]
Yesterday, I got my first Fire cape ever, with the wonderful time of 1:23:45.60.
Unfortunately, the !pb command shows this as 83:45.60
For more consistency (and a little flex) it would be nice if the !pb command also shows the pb times in the format of hh:mm:ss.t

**Comment by Nightfirecat** [comment, marked as answer]
I've promoted this to an issue for development: #15122
